google-cloud-kms/lib/google/cloud/kms/v1/doc/google/iam/v1/options.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Iam
+    module V1
+    end
```

Comment:
Missing definition in `options.rb`.

Reply:
@dazuma Did you see this? Also in [pubsub](https://github.com/googleapis/google-cloud-ruby/pull/3588).

Reply:
We decided offline to just keep an eye on this situation, and investigate if it happens again.